### PR TITLE
Update Sass documentation

### PR DESCRIPTION
**Pull request by irundaia**

### CODE REVIEW

File: documentation/manual/detailedTopics/assets/AssetsSass.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ h1 {
 }
 ```
 
-The Sass file outlined above, will be compiled into `public/stylesheets/main.css`. This file can be used in your template as any regular public asset:
+the Sass file outlined above, will be compiled into `public/stylesheets/main.css`. Hence, this file can be used in your template as any regular public asset:
```
Comment (benmccann) [inline documentation/manual/detailedTopics/assets/AssetsSass.md]: I think "The" was more appropriate

**Comment by benmccann** [comment]
lgtm. just one super minor comment

**Comment by irundaia** [comment]
Fair enough. Since I'm not a native English speaker, you're probably right ;)

**Comment by benmccann** [comment]
Thanks!

**Pull request merged**